SocketRocket/SRWebSocket.m

```diff
@@ -1383,6 +1383,16 @@ - (void)_sendFrameWithOpcode:(SROpCode)opcode data:(id)data;
     
     [self _writeData:frame];
 }
+static BOOL evaluateServerTrust(SecTrustRef serverTrust) {
+    BOOL isValid = NO;
+    SecTrustResultType result;
+
+    OSStatus status = SecTrustEvaluate(serverTrust, &result);
+    
+    isValid = ((result == kSecTrustResultUnspecified || result == kSecTrustResultProceed) && status == 0);
```

Comment:
If you're pinning a certificate that is in the user's trust store wouldn't that return a `SecTrustResultProceed`?